app/src/main/java/ch/epfl/cs311/wanderwave/ui/screens/BeaconScreen.kt

```diff
@@ -131,48 +160,91 @@ fun BeaconInformation(location: Location) {
 }
 
 @Composable
-fun SongList(beacon: Beacon) {
+fun SongList(beacon: Beacon, navigationActions: NavigationActions) {
   HorizontalDivider()
   Text(
       text = stringResource(R.string.beaconTracksTitle),
       style = MaterialTheme.typography.displayMedium,
       modifier = Modifier.testTag("beaconTracksTitle"))
-  LazyColumn { items(beacon.tracks) { TrackItem(it) } }
+  LazyColumn { items(beacon.profileAndTrack) { TrackItem(it, navigationActions) } }
 }
 
 @Composable
-internal fun TrackItem(track: Track) {
+internal fun TrackItem(
+    profileAndTrack: ProfileTrackAssociation,
+    navigationActions: NavigationActions
+) {
+  val scrollState = rememberScrollState()
+  val scope = rememberCoroutineScope()
+  val slowScrollAnimation: AnimationSpec<Float> = TweenSpec(durationMillis = 5000, easing = { it })
```

Comment:
Might be worth having things such as `durationMillis` be setup as global constants, I think it'll help us have a coherent interface in the long run

Reply:
Same thing for the two delays below